straxen/plugins/peaklet_processing.py

```diff
@@ -108,13 +108,26 @@ def setup(self):
         self.to_pe = straxen.get_correction_from_cmt(self.run_id,
                                        self.config['gain_model'])
 
+        # Check config of `hit_min_amplitude` and define hit thresholds
+        # if cmt config
+        if (isinstance(self.config['hit_min_amplitude'], tuple) and 
+            len(self.config['hit_min_amplitude'])==3 and 
+            type(self.config['hit_min_amplitude'][0]==str) and
+            type(self.config['hit_min_amplitude'][1]==str) and
+            type(self.config['hit_min_amplitude'][0]==bool)):
```

Comment:
Can we please add to CMT a function called `is_cmt_option` I think this would make our life much easier at several places. The function would look like:
```
def is_cmt_option(config):
    is_cmt = (isinstance(config, tuple) 
               and len(config)==3 
               and isinstance(config[0], str) 
               and isinstance(config[1], (str, int, float))
               and isinstance(config[2], bool))
    return is_cmt
```
Some additional comments: I think in that more general case the second entry can be either a string, integer or float. I think the index of your last test is supposed to be 2. Can you test my example before implementing? Not sure if I got the structure of the settings right.

Reply:
And please add a simple test. Something like which catches if we change the CMT option structure. 
```
def test_is_cmt_option():
    dummy_option
    assert is_cmt_option(dummy_option), 'Structure of CMT options changed!'
```

Reply:
Thanks for pointing out such a big mistake! Let me try your idea and do some retest.

Reply:
It is done in [commit fac7d93](https://github.com/XENONnT/straxen/pull/543/commits/fac7d935fbae4e5d736aa8424c6c2723db1291b9). Would you please take a look when you have time?